prometheus/README.md

```diff
@@ -55,13 +57,13 @@ you may need to use [Ngrok](https://ngrok.com/) to make HTTP requests to your Pr
 
 :::
 
-After installing Ngrok, run:
+After [installing Ngrok](https://ngrok.com/download), run the following in a separate terminal:
```

Comment:
Where is Ngrok installed?  Local machine?  Docker image?
Run in the local terminal?  terminal in the running image?  Which image?